src/components/AddMovie.jsx

```diff
@@ -1 +1,100 @@
-// implement AddMovie component here
+import React from 'react';
+import propTypes from 'prop-types';
+import AddMovies2 from './AddMovies2';
+
+// import AddMovies2 from './AddMovies2';
+
+class AddMovie extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+
+    this.state = {
+      title: '',
+      subtitle: '',
+      imagePath: '',
+      storyline: '',
+      rating: 0,
+      genre: 'action',
+    };
+  }
+
+  handleClick() {
+    const { onClick } = this.props;
+    onClick(this.state);
+    this.setState = ({
+      title: '',
+      subtitle: '',
+      imagePath: '',
+      storyline: '',
+      rating: 0,
+      genre: 'action',
+    });
+  }
+
+  handleChange({ target }) {
+    const { id, value } = target;
+
+    this.setState({
+      [id]: value,
+    });
+  }
+
+  render() {
+    const { title, subtitle, imagePath, storyline } = this.props;
+    const { rating, genre } = this.state;
```

Comment:
o seu componente AddMovie não recebe essas props, tudo isso esta no state
```suggestion
    const { title, subtitle, imagePath, storyline, rating, genre } = this.state;
```